mkdocs.yml

```diff
@@ -210,15 +206,7 @@ nav:
         - Jenkins: integration/jenkins.md
         - Terraform: integration/terraform.md
         - Synthetics for CI/CD:
-          - Overview: synthetics/ci-cd/overview.md
-          - Jenkins: synthetics/ci-cd/jenkins.md
-          - Travis CI: synthetics/ci-cd/travis-ci.md
-          - Circle CI: synthetics/ci-cd/circle-ci.md
-          - GitHub Actions: synthetics/ci-cd/github-actions.md
-          - Bitbucket Pipelines: synthetics/ci-cd/bitbucket-pipelines.md
-          - GitLab CI/CD: synthetics/ci-cd/gitlab-ci-cd.md
-          - Vercel: synthetics/ci-cd/vercel.md
-          - Netlify: synthetics/ci-cd/netlify.md
+          - GitHub Actions: synthetics/ci-cd/ci-cd-overview.md
```

Comment:
@HSSalman Can we change ci-cd-overview.md to overview.md? This way, we won’t have to update the links in other places where we might have referred to it, such as in old blog posts or in SC.

Reply:
Done